.github/CODEOWNERS

```diff
@@ -1,41 +1,21 @@
 # Global Owners
-* @nickboldt @ibuziuk @l0rd @sparkoo
+* @l0rd @ibuziuk @vinokurig @nickboldt 
 
-# dockerfiles
-dockerfiles/** @nickboldt
-
-# assembly
-assembly/** @nickboldt
-assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/** @ibuziuk @nickboldt @themr0c
-
-# core
-core/** @nickboldt
+# che.properties (quasi-documentation)
+# Fionn Kelleher (@osslate) is the replacement for Fabrice, but he doesn't seem to have rights to this repo, so comment this out for now
+# assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/** @osslate 
```

Comment:
do we really need to the comment?

Reply:
well, if we want docs team to eventually have access to che.properties file again, then this is a useful message to the future. :D